### PR TITLE
Add Subscriptions tab to User page in Admin

### PR DESCRIPTION
**Pull request by igorbp**
It adds a simplified version of the Subscription list to the user's show page. This is intended to make it easier to list user's subscriptions when accessing their pages.

users-subscriptions

### CODE REVIEW

File: app/views/spree/admin/shared/_subscription_tab.html.erb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <% if can? :admin, SolidusSubscriptions::Subscription %>
-  <%= tab :subscriptions, icon: 'repeat' %>
+  <%= tab :subscriptions, icon: 'repeat', match_path: '/subscriptions' %>
```
Comment (kennyadsl) [inline app/views/spree/admin/shared/_subscription_tab.html.erb:2]: Is this related to this PR? If yes, I'd use a separate commit that explains how they are related, otherwise a new PR.

Comment (igorbp) [reply app/views/spree/admin/shared/_subscription_tab.html.erb:2]: This is necessary to avoid having multiple options selected in the menu. I left it in the same PR to avoid having a known bug in a revision that could be visited in the future.

image

Comment (kennyadsl) [reply app/views/spree/admin/shared/_subscription_tab.html.erb:2]: I think a separate commit would be ideal for that change, but this is not blocking.

**Review by kennyadsl — APPROVED** [review]

**Review by elia — APPROVED** [review]
